app/controllers/katello/api/v2/content_view_filter_rules_controller.rb

```diff
@@ -122,5 +119,28 @@ def process_errata_ids(select_all_params)
         []
       end
     end
+
+    private def create_rules
```

Comment:
didn't even realize this was an option :+1: even though i think it looks kinda weird, we should do this whenever we have private methods